sitecore-xp0-sxa-horizon/README.md

```diff
@@ -0,0 +1,7 @@
+# Sitecore 10 XP0 with JSS Example
```

Comment:
Can we update this README to Horizon specific text

Reply:
How is this 
# Sitecore 10 XP0 with Horizon Example

An example repo for running Sitecore 10 XP0 with Horizon topology on Docker.

The build will layer in SPE,SXA and Horizon

Note that the override scales down the CD instance to 0 to save some CPU/RAM for local development purposes.